src/app/clusters/diagnostic-logs-server/Upgrading.md

```diff
@@ -0,0 +1,41 @@
+### DiagnosticLogsCluster API Changes
+
+The Diagnostic Logs cluster is a device-wide singleton. There is no need to
+manage delegates on a per-endpoint basis. To reflect this, the API has been
+simplified.
+
+What Changed
+
+#### SetDelegate() API Updated
+
+Old API (deprecated):
+
+```
+chip::app::Clusters::DiagnosticLogs::DiagnosticLogsServer::SetDiagnosticLogsProviderDelegate(endpointId, delegate);
```

Comment:
This seems untrue: the old and new code require another `Instance()` in the call. Should we correct it?